api-lib/gql/mutations.ts

```diff
@@ -533,3 +533,15 @@ export async function insertInteractionEvents(
   );
   return;
 }
+
+export async function insertActivity(
```

Comment:
same comment here as in `api-lib/gql/queries`... actually, in this case, the mutation is so straightforward i wouldn't bother splitting it out into a separate function at all